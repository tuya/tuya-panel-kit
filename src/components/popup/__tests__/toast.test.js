import React from 'react';
import { View } from 'react-native';
import renderer from 'react-test-renderer';
import Popup from '../index';

describe(' Popup Toast', () => {
  it('basic render', () => {
    const component = renderer.create(
      Popup.toast({
        message: 'I am Toast',
      })
    );
    expect(component).toMatchSnapshot();
  });
});
