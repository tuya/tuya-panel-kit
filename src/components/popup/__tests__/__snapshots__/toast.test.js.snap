// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Popup Toast basic render 1`] = `null`;
